### PR TITLE
sci-misc/boinc: New ebuild boinc-7.6.31.ebuild

### PR DESCRIPTION
**Pull request by Yamakuzure**
Gentoo-Bug: 577490

Package-Manager: portage-2.2.28

@SoapGentoo @jlec I've squashed the commits, hopefully it is all clean now.

### CODE REVIEW

**Comment by SoapGentoo** [comment]
Dear Sven,
_NEARLY_ perfect :stuck_out_tongue:. You've merged the master back in, instead of rebasing. Leave it as is, I'll just cherry-pick the commit. Message and format are perfect. @jlec please have the last word.

**Comment by idella** [comment]
"${ED}"/etc
I keep seeing this.  There are NO prefix keywords so there is no need for this to be set to ED.  Please reset to 
"${D}"etc
Also note the absence of the trailing slash with  "${D}"

**Comment by Yamakuzure** [comment]
Hello Ian,

thank you for your comment!

Am Mittwoch, 16. März 2016, 03:31:51 CET schrieb Ian Delaney:
> "${ED}"/etc
> I keep seeing this.  There are NO prefix keywords so there is no need for
> this to be set to ED.  Please reset to "${D}"etc
> Also note the absence of the trailing slash with  "${D}"

I am a bit confused right now.
The Gentoo development guide says, that the "D" variable is the path to the 
temporary install directory, the default value is "${PORTAGE_BUILDDIR}/image". 

But if I understand you correctly, the documentation is out of date, and the 
default value is "${PORTAGE_BUILDDIR}/image/" with a trailing slash instead, 
right?

However, I left the ED as is, as the Prefix-Project documentation suggests to 
use ${ED} instead of ${D%/}${EPREFIX}, which would be the same.

So this is no longer the case?

Thank you for your help!

Sven

**Comment by idella** [comment]
well, both work.  D has NO trailing slash and they expand to the same thing. The point is why are you guided by the Prefix-Project documentation? 
KEYWORDS="~amd64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
You have no prefix keywords therefore imo it is not an issue. With no intent to ever have prefix keywords, I see  "${D}"etc as the better selection, but  they expand to the same thing.

**Comment by jlec** [comment]
@Yamakuzure There is no harm in using ED as on non-prefix installations ED == D. In fact it is even good because later addition of prefix keywords only require KEYWORD change no alteration of the ebuild code.

File: sci-misc/boinc/boinc-7.6.31.ebuild [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (jlec) [inline sci-misc/boinc/boinc-7.6.31.ebuild]: Please use EAPI=6 if possible.

Comment (SoapGentoo) [inline sci-misc/boinc/boinc-7.6.31.ebuild]: @jlec we are currently in the process of co-ordinating with Ryan in order to have wxwidgets.eclass be `EAPI=6` compliant.

Comment (jlec) [inline sci-misc/boinc/boinc-7.6.31.ebuild]: @SoapGentoo ah okay.

**Comment by SoapGentoo** [comment]
Ok, merged, thanks Sven.

**Comment by Yamakuzure** [comment]
@idella : Sorry for the late reply, I was on vacation for two weeks, and cut from the internet. (which was not planned)

> well, both work. D has NO trailing slash and they expand to the same thing. The point is why are
> you guided by the Prefix-Project documentation?

I am not, I just looked it up. ED was used in the ebuilds before I took over the proxy maintainership, that's all. :-)

**Pull request closed without merging**